### DISCUSSION TITLE
why is there a element style codes?

### DISCUSSION THREAD

**Discussion by S4K01** [community/community#64818] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

hi, I am new at GitHub and I wanted to use the git page so I did all that it needed but when I finally used the page there was a strange 
element style codes so my website was different but when I copied the page link and sent it to my mobile it was normal and there 
Weren't any of the strange codes but it is still on my PC I don't know if it will disappear if I open the link on a different PC I will put my files and my page link 

my files
https://github.com/S4K01/work

my page link
https://s4k01.github.io/work/layout1.html

**Comment by arafajs** [comment, marked as answer]
- Make sure you are viewing the actual deployed website, not just the source code. The URL should be something like https://yourusername.github.io/yourrepositoryname, not just the GitHub repository page.
- Try doing a hard refresh in your browser (Ctrl+Shift+R in most browsers) to clear the cache. Sometimes cached code can cause issues.
- Double check that you don't have any raw HTML, JavaScript, or CSS code accidentally left in your source files that is getting rendered. Make sure all code is in the proper files (.html, .css, .js)
- Try viewing the page in another browser or incognito mode. Sometimes extensions or browser customization can affect rendering.
- Clear your browser cache and cookies. Old cached files could be interfering.
- Make sure your source code is formatted properly and all files are linked correctly. Errors like missing closing tags could display oddly.
- If you continue to see code on your PC but not mobile, it may be an issue with your local setup rather than the actual site. Try viewing on another computer to confirm.
- You can delete your published site and republish a fresh version.

**Reply by S4K01** [reply]
Thanks for your help actually the website and everything was good when I opened it on another pc I think the main problem was in GitHub I don't know if it is but thanks
